.devcontainer/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM rocker/verse:devel
 RUN apt-get update
 RUN apt-get install -y libxml2-dev libssl-dev gpg
 # texlive
-RUN apt-get install -y libqpdf26 qpdf ghostscript
+RUN apt-get install -y libqpdf28 qpdf ghostscript
+# RUN tlmgr install xcolor  # (more latex libs would be required to include pdf related checks)
 RUN install2.r --error devtools rJava RUnit zoo qpdf languageserver
+RUN installGithub.r r-lib/revdepcheck
```

Comment:
this is likely not reproducible, as it would use the source on the main branch, right ?

Reply:
The only tag they have is more than 3 years old though, I doubt it's worth to fix it at this point. Their README also suggests that the package should be on CRAN eventually.